crawl-ref/source/dat/strings/de/mutations.txt

```diff
@@ -1185,13 +1185,13 @@ You have a snake-like lower body.
 Ihr habt einen schlangenartigen Unterkörper.
 %%%%
 You can use your snake-like lower body to constrict enemies.
-Ihr könnt Euren schlangenartigen Unterkörper einsetzen, um Feinde zu umschlingen.
+Ihr könnt Euren schlangenartigen Unterkörper einsetzen, Eure Feinde zu umschlingen.
```

Comment:
This doesn't quite work.

Ihr könnt Euren schlangenartigen Unterkörper einsetzen, um Feinde zu umschlingen. (That's correct but meh.)
Ihr könnt Euren schlangenartigen Unterkörper dazu einsetzen, Feinde zu umschlingen. (This is better.)

Reply:
done